modules/nw-multus-dummy-device-object.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-multus-dummy-device-object_{context}"]
+= Configuration for a dummy device additional network
+
+The following object describes the configuration parameters for the dummy CNI plugin:
+
+.Dummy device CNI plugin JSON configuration object
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
+
+|`name`
+|`string`
+|The value for the `name` parameter you provided previously for the CNO configuration.
```

Comment:
This is arbitrary but I think the recommended value is useful (as it may appear this way in logging), nicely done.